src/transformStateWithClones.js

```diff
@@ -6,8 +6,34 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  let stateCopy = { ...state };
+  const stateCopyActions = [];
```

Comment:
Try to make up better name, for example `stateHistory`. Because now It looks like array of actions